hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -346,7 +346,7 @@ private void notifyWithExceptionWhenTargetIsNull() {
 
     @Override
     public void sendResponse(Operation op, Object response) {
-        if (!RESPONSE_RECEIVED.compareAndSet(this, FALSE, TRUE)) {
+        if (RESPONSE_RECEIVED.getAndSet(this, TRUE)) {
```

Comment:
why did you change this? The compareAndSet is more clear since it is more obvious you get a one time shot.